interp/vm.go

```diff
@@ -1085,17 +1076,23 @@ func (p *interp) callBuiltin(builtinOp compiler.BuiltinOp) error {
 		cmd.Stdout = p.output
 		cmd.Stderr = p.errorOutput
 		_ = p.flushAll() // ensure synchronization
-		err := cmd.Run()
+		err := cmd.Start()
+		if err != nil {
+			// could not start the shell
+			p.printErrorf("%v\n", err)
+			p.replaceTop(num(-1.0))
+			return nil
```

Comment:
Returning will stop executing bytecodes, right? I'm pretty sure we don't want to do that here. Could we see if we can add a test case for this (and then remove the return)?

Reply:
I seem to remember starting at this. I don't think it will stop _all_ execution, it will just stop the `callBuiltin` function. This is what `callBuiltin` returns at its conclusion.

The goal is to skip the rest of the code in the `case` block. What if I switch the return to `break`? That is less jarring than the `return nil`. I will study the existing interp tests. I seem to remember some test cases there for missing shell commands.

By the way, I split the `exec.Cmd.Run()` into `.Start()` and `.Wait()` so I could separate a starting error and an error result on child process exit. That is because the `cmdExitCode()` assumes the process started.

Reply:
Ah, sorry -- I thought this was in `execute` instead of `callBuiltin`. No, `return nil` is fine given the context (I was looking at this in GitHub's diff viewer, not my IDE).

Re the separation between `Run` and `Start` -- yep, makes sense, thanks.